contrib/dev/Vagrantfile

```diff
@@ -99,19 +105,31 @@ def populate_os_config(os, ip)
       address: '192.168.10.10',
     },
   ]
+  os.security_groups = [ 'skydive' ]
+end
+
+def fedora_box(vm, type)
+  major = "27" 
+  minor = "1.6"
+  vm.box = "fedora/#{major}-cloud-base"
+  vm.box_url = "https://download.fedoraproject.org/pub/fedora/linux/releases/#{major}/CloudImages/x86_64/images/Fedora-Cloud-Base-Vagrant-#{major}-#{minor}.x86_64.vagrant-#{type}.box"
 end
 
 Vagrant.configure(2) do |config|
   config.vm.define "dev" do |dev|
     dev.vm.hostname = "dev"
     dev.vm.synced_folder ".", "/vagrant", disabled: true
-    dev.vm.synced_folder SYNCED_FOLDER, "/home/vagrant/src/github.com/skydive-project/skydive", disabled: false, :type => "nfs", :nfs_udp => false
+    if SKYDIVE_SYNC_FOLDER then
+      dev.vm.synced_folder SKYDIVE_HOST, SKYDIVE_GUEST, type: "nfs", nfs_udp: false, user: "vagrant", group: "vagrant"
```

Comment:
This does not work on the machines we use to build and upload the "official" images (they use Linux and libvirt)